jetty-webapp/src/main/java/org/eclipse/jetty/webapp/StandardDescriptorProcessor.java

```diff
@@ -1113,6 +1113,8 @@ public void visitErrorPage(WebAppContext context, Descriptor descriptor, XmlPars
             code=Integer.valueOf(error);
 
         String location = node.getString("location", false, true);
+        if (!location.startsWith("/"))
+            throw new IllegalStateException("The value of the location must have a leading '/'. Missing leading '/' for location:" + location);
```

Comment:
The exception message seems duplicate, as you're saying the same thing twice.
Can you just change to 
```
throw new IllegalStateException("Missing leading '/' for location: " + location);
```
Note the space after the semicolon too.
Otherwise the patch looks good.